src/index.ts

```diff
@@ -1,38 +0,0 @@
-export { Arc, IContractInfo } from './arc'
```

Comment:
We still want to have an index file for the `src` folder to support the following syntax `import { Something } from '@daostack/client'`

I do think that we should utilize the `export * from './dao'` syntax more though.